core/station/impl/src/services/external_canister.rs

```diff
@@ -669,13 +669,6 @@ impl ExternalCanisterService {
                     )?,
                 ExternalCanisterChangeCallRequestPoliciesInput::OverrideSpecifiedByExecutionValidationMethodPairs(calls) => {
                     for call in calls {
-                        let initial_policy_ids = self.request_policy_repository
```

Comment:
Is there a reason to inline the let binding?

Reply:
mostly to make `OverrideSpecifiedByExecutionValidationMethodPairs` and `OverrideSpecifiedByExecutionMethods` consistent, no strong preference through